main/acle.md

```diff
@@ -9049,6 +9070,61 @@ Interleave elements from halves of each pair of quadword vector segments.
    svuint8_t svzipq2[_u8](svuint8_t zn, svuint8_t zm);
    ```
 
+### SVE2 maximum and minimum absolute value
+
+#### FAMAX
+
+Floating-point absolute maximum (predicated).
+``` c
+  // Variants are also available for: _f32 and _f64
+  svfloat16_t svamax[_f16]_m(svbool_t pg, svfloat16_t zn, svfloat16_t zm);
+  svfloat16_t svamax[_f16]_x(svbool_t pg, svfloat16_t zn, svfloat16_t zm);
+  svfloat16_t svamax[_f16]_z(svbool_t pg, svfloat16_t zn, svfloat16_t zm);
+
+  svfloat16_t svamax[_n_f16]_m(svbool_t pg, svfloat16_t zn, float16_t zm);
+  svfloat16_t svamax[_n_f16]_x(svbool_t pg, svfloat16_t zn, float16_t zm);
+  svfloat16_t svamax[_n_f16]_z(svbool_t pg, svfloat16_t zn, float16_t zm);
+```
+
+#### FAMIN
+
+Floating-point absolute minimum (predicated).
+``` c
+  // Variants are also available for: _f32 and _f64
+  svfloat16_t svamin[_f16]_m(svbool_t pg, svfloat16_t zn, svfloat16_t zm);
+  svfloat16_t svamin[_f16]_x(svbool_t pg, svfloat16_t zn, svfloat16_t zm);
+  svfloat16_t svamin[_f16]_z(svbool_t pg, svfloat16_t zn, svfloat16_t zm);
+
+  svfloat16_t svamin[_n_f16]_m(svbool_t pg, svfloat16_t zn, float16_t zm);
+  svfloat16_t svamin[_n_f16]_x(svbool_t pg, svfloat16_t zn, float16_t zm);
+  svfloat16_t svamin[_n_f16]_z(svbool_t pg, svfloat16_t zn, float16_t zm);
+```
+
+### SVE2 lookup table
+
+#### LUTI2
+
+Lookup table read with 2-bit indices.
+```c
+  // Variant is  also available for: _u8
+  svint8_t svluti2[_s8](svint8_t table, svuint8_t indices, uint64_t imm0_3);
+
+  // Variant are also available for: _u16, _f16 and _bf16
+  svint16_t svluti2[_s16]( svint16_t table, svuint8_t indices, uint64_t imm0_7);
+```
+
+#### LUTI4
+
+Lookup table read with 4-bit indices.
+```c
+  // Variant is also available for: _u8
+  svint8_t svluti4[_s8](svint8_t table, svuint8_t indices, uint64_t imm0_1);
+
+  // Variant are also available for: _u16, _f16, _bf16
+  svint16_t svluti4[_s16](svint16_t table, svuint8_t indices, uint64_t imm0_7);
+  svint16_t svluti4[_s16]_x2(svint16x2_t table, svuint8_t indices, uint64_t imm0_7);
+```
+
```

Comment:
I am not sure if I understand naming conventions correctly here, but shouldn't svluti... instrisics have _lane suffix since they use immediate index ?

Reply:
Done

Reply:
The analogy to other intrinsics is only approximate. Most _lane intrinsics use the index to select a lane within each 128-bit segment. The only existing exceptions I'm aware of are `svdup_lane/`svdupq_lane`, which index element sized or quadword sized chunks over the whole register, and return an all-zero register if the index is out of range.

In all of these cases, the indexed element has a fixed size, and is broadcast in parallel across each lane of the operation.

On the other hand, `svluti` operates on a chunk whose size scales with the length of the vector, and each part of the indexed register is used for just one element of the operation. So this is a very different scenario to any of the existing SVE `_lane` intrinsics.

A more thorough basis for the decision would be to note that the existing SME luti intrinsics already have `_lane` suffixes, and the AdvSIMD intrinsics are proposed to have `_lane` intrinsics as well, with the `_lane`/`_laneq` distinction being essential to identifying the input parameter type.